crates/agent/Cargo.toml

```diff
@@ -36,6 +36,7 @@ context_server.workspace = true
 convert_case.workspace = true
 db.workspace = true
 editor.workspace = true
+extension_host.workspace = true
```

Comment:
@bennetbo We shouldn't have the `agent` crate depend on `extension_host`.

We have a separate `ExtensionEvents` type that can be used for receiving extension-related events by depending on the much lighter-weight `extension` crate:

https://github.com/zed-industries/zed/blob/f85fd0c3b692cad756ef1c3b31221a0d0c2cd502/crates/extension/src/extension_events.rs#L32-L35

Reply:
Sounds good, is there any existing way to get the extension manifest from the extension id, other than using the extension_host crate?